validator/sawtooth_validator/execution/context_manager.py

```diff
@@ -320,6 +320,8 @@ def __init__(self, database, state_delta_store):
 
         self._address_regex = re.compile('^[0-9a-f]{70}$')
 
+        self._namespace_regex = re.compile('^[0-9a-f]{0,70}$')
```

Comment:
@boydjohnson thought, "hmmm....I have a problem, I think I'll use a regular expression to fix it"....now he has two problems :)

or `([0-9a-f]{2}){0,35}`

Reading regular expressions reminds me of the old obfuscated C coding contests.